src/style.css

```diff
@@ -1,3 +1,91 @@
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+ul,
+ol {
+  padding-left: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+button {
+  cursor: pointer;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  text-transform: uppercase;
+  line-height: 1.5;
+  letter-spacing: 0.02em;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+}
+
+.list {
+  list-style-type: none;
+}
+
+.link {
+  text-decoration: none;
+}
+
+:root {
+  --pressed: black;
+  --accent-gray: #00acdc;
+}
+
+.header {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin-right: 50px;
+  margin-left: 50px;
+  display: flex;
+  justify-content: space-between;
```

Comment:
Don't fix container size (if there is no such a requirement). In this case, setting explicit margins might not be necessary if you want the container to be sized by its content. If there's no requirement for fixed margins, consider using padding within the container or setting max-width instead.